.github/workflows/test.yml

```diff
@@ -77,8 +77,7 @@ jobs:
       - name: Test Integration
         env:
           NEXTEST_PROFILE: integration
-          INTEGRATION_TEST_SEQUENCER_VERSION: 02
-        run: cargo nextest run --locked --release --all-features --verbose --nocapture
+        run: cargo nextest run --locked --release --all-features --verbose --nocapture --test integration smoke
```

Comment:
What exactly does this do now?

Reply:
so the there are two tests inside integration: smoke and upgrade test. This one just runs smoke test for V2 version.

Reply:
as we are not doing any upgrade for the  native demo now

Reply:
test.yaml changed a lot last week. In the new version you probably only need to remove the `02` from `matrix.version`